src/app/page.tsx

```diff
@@ -141,11 +141,10 @@ const ActionBox: ComponentType = () => (
 
 export default function Home() {
 	return (
-		<main>
+		<>
```

Comment:
this is bound to get messy for other pages sharing the same layout, with nested mains, etc. no?

Reply:
Yes I think so that's why I moved the main element to the root layout
Not header, not footer, it's main - we could narrow what "main" means in our dashboard context more but "not header, not footer" is a good start I think

layouts nest

Reply:
my concern here is others not using a fragment as the root of the page, then it's a "double main" etc.

Reply:
ah i see i see - updated

agree. worse to have multiple mains, or terrible markup nesting than to not have one at all.

Reply:
yeah. I'm not loving this new layout vs page thing in nextjs